### ISSUE TITLE
Outlook crashes When NVDA is used to view the calendar reminders window

### ISSUE THREAD

**Issue by Ednunp** [nvaccess/nvda#8601]
Hello. I am experiencing a crash in Outlook when viewing the calendar reminders pop-up window, please see below.

Steps to reproduce:
1. Ensure you have at least 2 meetings in the calendar which are about to start or have already started. I cannot always recreate this issue if there is only 1 meeting.

2. Outlook should pop up a window titled for example, 2 reminders.

3. Move to this window and attempt to read elements using the arrow or tab keys.

Actual behavior:
Outlook stops responding, crashes and restarts.

Expected behavior:
Outlook should allow navigation of the window.

System configuration:
windows Version	10.0.16299 Build 16299
MS Outlook 16.0.91226.229

NVDA is installed, version Alpha 15781.

additional info:
I have tried restarting the computer and NVDA, to no effect. Whilst the issue does not always appear, it has only started since I recently updated Outlook but I'm afraid I cannot tell what build I was on before. I updated Outlook to resolve a crash when navigating the Calendar as was reported last year.

**Comment by ehollig** [comment]
It is possible that this is an Outlook bug. @Ednunp what exact version of Office are you running now, and is it possible to include an NVDA log set to debug when the crash occurs or a crash dump? Please see [Log Files and Crash Dumps](https://github.com/nvaccess/nvda/wiki/LogFilesAndCrashDumps) for more information on how to do this.

**Comment by Ednunp** [comment]
HI. I have included the outlook version number above, it's Office 2016 but I will attempt to retreave a log and dump file next time this happens. Thanks.

**Comment by michaelDCurran** [comment]
I cannot reproduce this with 16.0.10325.20082.
Please go to File -> Office Account -> Update Options, and check for updates.
If you are able to update, please do and then test again.

**Comment by Ednunp** [comment]
HI. Really silly question, but is your version of Outlook newer than the one I have which is 16.0.91226.229? it seems like because of the 1, it might be but not sure. Either way, I can't update mine, it says it's up to date. Are you running on internals?

**Comment by michaelDCurran** [comment]
Mine is certainly ahead of yours. I had the 16.0.9xxx some time ago.

I am on Office 365 Business, monthly release channel.

Not Office Insider at the moment.

In File -> Office Account -> Office updates grouping, you can use object 
navigation to see the channel.

**Comment by Ednunp** [comment]
Good tip re the updates channel, I never realised 365 could do that or that my company would allow it. I used the channel selector program and it seems to have updated it to a later version and no more errors. Thank you. Happy for the issue to be closed. Can you select the update channel with the regular user i.e. not business purchase of 365? strikes me it would be good to have both on the monthly channel releases.

**Comment by Ednunp** [comment]
Also FYI I was not able to see which channel I was on with object nav. Can you describe the steps to do this please if not too much trouble?

**Comment by Adriani90** [comment]
@Ednunp I had a similar issue some months ago but after a new NVDA release it was solved. I don't know exactly which one it was though. Are you still having this problem with NVDA 2019.1.1 and most current version of Outlook 2016? I cannot reproduce it anymore.

**Comment by Ednunp** [comment]
No I thought this one was closed as the issue was fixed a while back.

**Comment by Adriani90** [comment]
Oh ok, thanks for reporting back so fast. I am closing this now as works for me.
